DataFormats/DetId/BuildFile.xml

```diff
@@ -1,6 +1,7 @@
 <use   name="boost"/>
 
 <use   name="DataFormats/Common"/>
+<use name="FWCore/Common"/>
```

Comment:
I'm not sure if data format package is allowed to depend on `FWCore/Common`.

Reply:
i donno... seems to work...

Reply:
ok, thx